apps/earn-protocol/app/server-handlers/portfolio-bulk-request/portfolio-bulk-request.ts

```
@@ -0,0 +1,111 @@
+    ])
+
+    const [positionsList, interestRatesResults] = allResults
+
+    const { config } = parseServerResponseToClient(systemConfig)
+
+    const vaultsDecorated = decorateCustomVaultFields({
+      vaults: vaultsList.vaults,
+      systemConfig: config,
+      decorators: {
+        arkInterestRatesMap: interestRatesResults.reduce((acc, curr) => {
+          return { ...acc, ...curr }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)